api/api-doc/storage_service.json

```diff
@@ -833,6 +833,35 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/repair_status/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Query the repair status and return when the repair is finished",
+               "type":"string",
+               "enum":[
+                  "RUNNING",
+                  "SUCCESSFUL",
+                  "FAILED"
+               ],
+               "nickname":"repair_async_status_until_finished",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"id",
+                     "description":"The repair ID to check for status",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type": "long",
```

Comment:
We should have made repair IDs UUIDs, not lonm but too late now.

What is returned if the repair is not recognized?  That can happen if the repair was completed before the request was sent.

Reply:
Maybe we should keep completed repairs for a while.

Reply:
Yes, I though about using UUID for repair id as well. My idea is to keep the old int based id for the old api caller. Use uuid based id with new api. 

A repair id completed successfully will be removed. Only failed repairs are stored. So, a repair id is not recognized, it will be SUCCESSFUL. Currently, we have a very simple check to see if the id is a valid one by checking if the id is beyond the next_repair_id.

Reply:
It may be successful, or non-existent.